app/controllers/albumsController.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable new-cap */
+
+const {
+  getAllAlbums,
+  getAlbumById,
+  getPhotosFromAlbum,
+  getPhotoFromAlbumByIds
+} = require('../services/albums');
+
+const showAllAlbums = (req, res) => {
+  console.log('----/albums----');
+
+  getAllAlbums()
+    .then(result => res.send(result.body))
+    .catch(error =>
+      res.status(500).send({ error: `There where errors getting albums ${JSON.stringify(error)}` })
+    );
+};
+
+const showAlbumById = (req, res) => {
+  const { id } = req.params;
```

Comment:
This const is not needed. You can just pass req.params.id to `getAlbumById`
Once you do that you can delete curly braces too. (And console.logs)